app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,7 @@
     <link rel="apple-touch-icon" href="/icon.png">
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+    <%= stylesheet_link_tag "application" %>
```

Comment:
We actually already have this just above on line 16. :) If you can remove please?

Reply:
Done, why did you add this line on my commit? :O